x-pack/plugins/security_solution/public/detection_engine/rule_management/components/translations.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const UNABLE_TO_FETCH_RULE_SNOOZE_SETTINGS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleManagement.ruleSnoozeBadge.error.unableToFetch',
+  {
+    defaultMessage: 'Unable to fetch snooze settings',
+  }
```

Comment:
It's a good point. I just wanted to avoid creating a folder with an `index.ts` file containing component. What if the translation moved inside the component's file?